cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1214,7 +1214,7 @@ protected CompositeFuture adminClientSecrets() {
         }
 
         Future<ReconciliationState> zkNetPolicy() {
-            return withVoid(networkPolicyOperator.reconcile(namespace, ZookeeperCluster.policyName(name), zkCluster.generateNetworkPolicy(pfa.isNamespaceAndPodSelectorNetworkPolicySupported(), operatorNamespace, operatorNamespaceLabels)));
+            return withVoid(networkPolicyOperator.reconcile(namespace, ZookeeperCluster.policyName(name), zkCluster.generateNetworkPolicy(true, operatorNamespace, operatorNamespaceLabels)));
```

Comment:
The `true` parameter should not be needed anymore, so I this you should remove it from the `generateNetworkPolicy` method ... not just here but also for all of the other places.